app/controllers/types_controller.rb

```diff
@@ -0,0 +1,41 @@
+class TypesController < ApplicationController
+  load_and_authorize_resource except: [:show]
```

Comment:
@VanjaJeftic do you have `show` method? If you do... 1) always write the method in the controller, even if it’s empty. That indicates to other people there’s something there. 2) why are you not loading/authorizing type? Why are you using `set_type`?

Reply:
@basti I realized that I made a mistake here.

Reply:
@basti I write that because in index method I have a line that doesn't use the authorized value of types (Type.all.order('created_at DESC').page(params[:page]).per(4)), and on triggering that line in view, on 'show' button click, 'show' dialog won't appear.